modules/viewing-the-details-of-a-pipeline-run.adoc

```diff
@@ -24,12 +24,15 @@ endif::[]
 The *Pipelines* page opens.
 . From the *Project* list, select the project whose pipeline runs you want to view.
 . For a pipeline that you want to see run details for, click *Expand* (image:images/rhoai-expand-icon.png[]).
-. From the *Runs* section, click the name of the run that you want to view the details of.
+. Click *View runs* on the row containing the pipeline version whose run logs you want to view.
 +
+The *Runs* page opens displaying a list of previously executed pipeline runs. 
+. Click the name of the run whose logs you want to view.
```

Comment:
> ...the run that you want to view logs for.

Same as previous RE: 'whose'/personifying.

Reply:
Fixed.

Reply:
Same as previous comment, as part of the implementation of the pipeline logs and pipeline versions functionality, the UI for viewing the details of a pipeline run has slightly changed, meaning that the previously documented steps no longer apply. 

Because of this, I changed the steps around a bit. In this module, the point is that we are just viewing the details of a run, rather than checking its logs.